expyfun/_utils.py

```diff
@@ -571,6 +572,8 @@ def get_config(key, default=None, raise_error=False):
     else:
         with open(config_path, 'r') as fid:
             config = json.load(fid)
+            if key is None:
+                return config
```

Comment:
this doesn't need to be indented (keep as little code in the context manager as necessary)